docs/user/guides/importing-private-repositories.rst

```diff
@@ -20,6 +20,11 @@ Here is how you set it up.
 
   Follow the steps below.
 
+.. warning::
+
+   If you have the :doc:`/commercial/single-sign-on` integration enabled, you cannot import projects manually.
+   This is because authorization to the documentation depends on a connection to your VCS provider which isn't established during manual import.
+   Please disable the VCS SSO integration to proceed with manual project import.
```

Comment:
Using "Please disable the VCS SSO..." sounds something trivial to do and without risks to me. I think it's because of the "Please". I'd use something more direct like "You have to disable VCS SSO if your organization requires to proceed..."

I don't know to explain myself, but the warning note looks inoffensive in the way it's written; but it has a big impact in reality.

Reply:
I agree, this is recommending a rather destructive action and users might not understand that this would kill authorization to projects. We should probably leave it at manual creation is impossible for these organizations.